source/features/prevent-duplicate-pr-submission.tsx

```diff
@@ -0,0 +1,29 @@
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let previousSubmission = Date.now() - 1000;
+
+function preventSubmit(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	if (Date.now() - previousSubmission < 1000) {
+		event.preventDefault();
+	}
+
+	previousSubmission = Date.now();
+}
+
+function init(): void {
+	delegate(document, '#new_pull_request button[type="submit"][aria-label$="for review"]', 'click', preventSubmit);
```

Comment:
This doesn't work?

```suggestion
	delegate(document, '#new_pull_request', 'submit', preventSubmit);
```

Reply:
Let me try that. I was doing submit on the button